### PR TITLE
Use the release AspNet feed

### PR DESCRIPTION
**Pull request by pranavkm**
Using aspnetvnext pulls in rc3 builds of some packages.

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @piotrpMSFT - the newest build of xunit uses RC3 builds of AspNet packages. Ideally we should remove that dependency but this would be a start.

File: NuGet.Config [hunk 1]
```diff
@@ -2,8 +2,7 @@
 <configuration>
   <packageSources>
     <clear/>
-    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
-    <add key="CI Builds (aspnetcidev)" value="http://myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetrelease/api/v3/index.json" />
```
Comment (eerhardt) [inline NuGet.Config:5]: Would it a good idea for this repo to just use the cli-deps feed?  https://dotnet.myget.org/F/cli-deps/api/v3/index.json

Comment (eerhardt) [inline NuGet.Config:5]: /cc @piotrpMSFT @Sridhar-MS @brthor @anurse

Comment (pranavkm) [inline NuGet.Config:5]: Sure. I could change the dependency to -rc2-\* to ensure it doesn't pick up newer packages.

**Pull request merged**